pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -429,46 +432,43 @@ func TestDump(t *testing.T) {
 		makeBasePod(t, nodeName, "test-1", "100m", "500", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 80, Protocol: "TCP"}}),
 		makeBasePod(t, nodeName, "test-2", "200m", "1Ki", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 80, Protocol: "TCP"}}),
 	}
-	tests := []struct {
+	test := struct {
 		podsToAssume []*v1.Pod
 		podsToAdd    []*v1.Pod
-	}{{ // two pod were assumed at same time. But first one is called Add() and gets confirmed.
+	}{ // two pod were assumed at same time. But first one is called Add() and gets confirmed.
 		podsToAssume: []*v1.Pod{testPods[0], testPods[1]},
 		podsToAdd:    []*v1.Pod{testPods[0]},
-	}}
+	}
 
-	for i, tt := range tests {
-		t.Run(fmt.Sprintf("case_%d", i), func(t *testing.T) {
-			logger, ctx := ktesting.NewTestContext(t)
-			ctx, cancel := context.WithCancel(ctx)
-			defer cancel()
-			cache := newCache(ctx, ttl, time.Second)
-			for _, podToAssume := range tt.podsToAssume {
-				if err := assumeAndFinishBinding(logger, cache, podToAssume, now); err != nil {
-					t.Errorf("assumePod failed: %v", err)
-				}
-			}
-			for _, podToAdd := range tt.podsToAdd {
-				if err := cache.AddPod(logger, podToAdd); err != nil {
-					t.Errorf("AddPod failed: %v", err)
-				}
-			}
+	logger, ctx := ktesting.NewTestContext(t)
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	cache := newCache(ctx, ttl, time.Second)
+	for _, podToAssume := range test.podsToAssume {
+		if err := assumeAndFinishBinding(logger, cache, podToAssume, now); err != nil {
+			t.Errorf("assumePod failed: %v", err)
+		}
+	}
+	for _, podToAdd := range test.podsToAdd {
+		if err := cache.AddPod(logger, podToAdd); err != nil {
+			t.Errorf("AddPod failed: %v", err)
+		}
+	}
 
-			snapshot := cache.Dump()
-			if len(snapshot.Nodes) != len(cache.nodes) {
-				t.Errorf("Unequal number of nodes in the cache and its snapshot. expected: %v, got: %v", len(cache.nodes), len(snapshot.Nodes))
-			}
-			for name, ni := range snapshot.Nodes {
-				nItem := cache.nodes[name]
-				if !reflect.DeepEqual(ni, nItem.info) {
-					t.Errorf("expect \n%+v; got \n%+v", nItem.info, ni)
-				}
-			}
-			if !reflect.DeepEqual(snapshot.AssumedPods, cache.assumedPods) {
-				t.Errorf("expect \n%+v; got \n%+v", cache.assumedPods, snapshot.AssumedPods)
-			}
-		})
+	snapshot := cache.Dump()
+	if len(snapshot.Nodes) != len(cache.nodes) {
+		t.Errorf("Unequal number of nodes in the cache and its snapshot. expected: %v, got: %v", len(cache.nodes), len(snapshot.Nodes))
+	}
+	for name, ni := range snapshot.Nodes {
+		nItem := cache.nodes[name]
+		if !reflect.DeepEqual(ni, nItem.info) {
+			t.Errorf("expect \n%+v; got \n%+v", nItem.info, ni)
+		}
+	}
+	if !reflect.DeepEqual(snapshot.AssumedPods, cache.assumedPods) {
```

Comment:
Please use cmp.Diff instead of DeepEqual. Same in some other parts in this file. If you want to make it out-of-scope in this PR, another PR to follow up this is OK.

Reply:
Ok, I will open a new PR to follow up this.